Sources/Proxy/Core/ViewProxy/index.js

```diff
@@ -253,9 +253,9 @@ function vtkViewProxy(publicAPI, model) {
 
   publicAPI.setAnimation = (enable) => {
     if (enable) {
-      model.renderWindow.getInteractor().requestAnimation('proxy');
+      model.renderWindow.getInteractor().requestAnimation(publicAPI);
     } else {
-      model.renderWindow.getInteractor().cancelAnimation('proxy');
+      model.renderWindow.getInteractor().cancelAnimation(publicAPI);
```

Comment:
@jourdain is that enough to make distinction between different viewproxys?

Reply:
yes I guess.